shared/js/l10n.js

```diff
@@ -1172,10 +1175,22 @@
 
       switch (type) {
         case 'json':
-          io.loadJSON(path, onJSONLoaded, sync);
+          if (source === 'bundle') {
```

Comment:
How about `app` instead of `bundle`?  At first, I thought this was referring to _localization bundles_, i.e. langpacks.